### ISSUE TITLE
Constant water walking

### ISSUE THREAD

**Issue by sefldron** [PrinceOfAmber/Cyclic#503]
Minecraft version & Mod Version:
1.12.1 cyclic v 1.8.2

Single player or Server:
Single

Describe problem (what you were doing / what happened):
Just wearing boots. Water walking without enchantment applied. Crafted a new pair of iron boots to test just to make sure and those allowed water walking as well. Removing boots while on water caused normal behavior until putting on any pair of boots. I'm also running 107 mods so if needed I will pastebin and link my log files if I find anything weird inside of them.

**Comment by sefldron** [comment]
Found one warn message. 
 Registry enchantment: object did not get ID it asked for. Then the name of the water walking enchantment Expected 76 got 88
Could be from silentlib registration for his enchantment registry concerning his enchantment token system with gem tools?
I understand his system to register enchants regardless of them actually having a crafting recipe for the tokens.
 Not saying that this is the issue but while searching using keywords in notepad I did not find anything else yet. Still looking though.
